apps/examples/testcase/ta_tc/media/utc/utc_media_mediarecorder.cpp

```diff
@@ -289,6 +291,82 @@ class RecorderTest : public MediaRecorderObserverInterface
 	std::condition_variable cvError;
 };
 
+static int get_file_size()
+{
+	int ret;
+	FILE *fp;	
+	fp = fopen(filePath, "r");
+	if (fp == NULL) {
+		meddbg("file open failed error : %d\n", errno);
+		return -1;
+	}
+	fseek(fp, 0, SEEK_END);
+	ret = ftell(fp);
+	fclose(fp);
+	return ret;
+}
+
+static void utc_media_MediaRecorder_setDuration_p(void)
+{
+	MediaRecorder mr;
+	unique_ptr<FileOutputDataSource> dataSource = unique_ptr<FileOutputDataSource>(new FileOutputDataSource(channels, sampleRate, pcmFormat, filePath));
+
+	auto observer = std::make_shared<RecorderTest>();
+	mr.create();
+	mr.setObserver(observer);
+	mr.setDataSource(std::move(dataSource));
+
+	TC_ASSERT_EQ("utc_media_MediaRecorder_setDuration", mr.setDuration(SECOND), RECORDER_OK);
+	
+	mr.prepare();
+	mr.start();
+	sleep(SECOND + 1);
+	mr.stop();
+	mr.unprepare();
+	mr.destroy();
+
+	int byte = 0;
+
+	switch (pcmFormat) {
+	case AUDIO_FORMAT_TYPE_S8:
```

Comment:
this is tc for media, please use audio_format_type_e in mediatype.h instead of pcmformat

Reply:
@Taejun-Kwon  I used audio_format_type_e in mediatype.h.